src/content/it/fundamentals/getting-started/_toc.yaml

```diff
@@ -0,0 +1,33 @@
+toc:
```

Comment:
We do not translate the TOC files, if we do, then any new content added will not show up in the translated version unless is explicitly added.  Please remove any `_toc.yaml` or `_book.yaml` files.